sycl/test/on-device/sub_group_algorithms/all_of.cpp

```diff
@@ -0,0 +1,72 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -I . -o %t.out
+// XUN: %HOST_RUN_PLACEHOLDER %t.out
```

Comment:
could you please  add TODO for disabled execution line

Reply:
done.